app/views/people/show.html.erb

```diff
@@ -27,6 +27,9 @@
   <div class="govuk-grid-column-two-thirds">
     <%= render partial: "biography", locals: { person: person } %>
 
+    <%= render partial: "current_roles", locals: { person: person} %>
+
     <%= render partial: "previous_roles", locals: { person: person } %>
   </div>
 </div>
+
```

Comment:
You're adding a empty line at the end of this file, which seems a bit redundant.